From 5263792a3c87e00c7d33cfa8502058f7615f5aca Mon Sep 17 00:00:00 2001
From: Bryce Harrington <bryce@bryceharrington.org>
Date: Thu, 12 Feb 2009 21:15:07 -0800
Subject: [PATCH] lt:  KPDL should produce comma, not period; add balticplus

As designed by LST 1582:2000 http://ims.mii.lt/klav/lithkeyb4.html
numpad KPDL key should produce comma character, but not a period
character.

Also add symbols for "Lithuania - Baltic+"
---
 symbols/lt |   32 ++++++++++++++++++++++++++++++--
 1 files changed, 30 insertions(+), 2 deletions(-)

diff --git a/symbols/lt b/symbols/lt
index 6defc44..d535da7 100644
--- a/symbols/lt
+++ b/symbols/lt
@@ -110,6 +110,8 @@ xkb_symbols "std" {
 
     key	<LSGT> {[	less,		greater,	endash		]};
 
+    include "kpdl(comma)"
+
     include "level3(ralt_switch)"
 };
 
@@ -158,9 +160,9 @@ xkb_symbols "ibm" {
     include "level3(ralt_switch)"
 };
 
-// LEKP and LEKPa layouts 1.0
+// LEKP, LEKPa and Baltic+ layouts 1.0 for Linux.
 // Copyright (c) 2007 Tautrimas Pajarskas
-// For more info visit http://lekp.info
+// For more info go to http://lekp.info
 //
 // LEKP and LEKPa layouts are licensed under the Creative Commons
 // Attribution-Noncommercial-Share Alike 3.0 License.
@@ -290,3 +292,29 @@ xkb_symbols "lekpa" {
     include "level3(ralt_switch)"
 };
 
+partial alphanumeric_keys modifier_keys
+xkb_symbols "balticplus" {
+    include "latin"
+    name[Group1]="Lithuania - Baltic+";
+    key <AE01>  { [   aogonek,      Aogonek,        1,      exclam ]    };
+    key <AE02>  { [    ccaron,       Ccaron,        2,          at ]    };
+    key <AE03>  { [   eogonek,      Eogonek,        3,  numbersign ]    };
+    key <AE04>  { [ eabovedot,    Eabovedot,        4,      dollar ]    };
+    key <AE05>  { [   iogonek,      Iogonek,        5,     percent ]    };
+    key <AE06>  { [    scaron,       Scaron,        6, asciicircum ]    };
+    key <AE07>  { [   uogonek,      Uogonek,        7,   ampersand ]    };
+    key <AE08>  { [   umacron,      Umacron,        8,    asterisk ]    };
+    key <AE09>  { [ parenleft, doublelowquotemark,  9 ] };
+    key <AE10>  { [ parenright, leftdoublequotemark, 0 ] };
+    key <AE11>  { [     minus,   underscore,    minus,  section ] };
+    key <AE12>  { [    zcaron,       Zcaron,    equal,        plus ]    };
+    key <CAPS>  { [ BackSpace ] };
+    key <NFER>  { [ backslash,         bar,   endash ] };
+    key <AD03>  { [         e,            E, EuroSign ] };
+    key <AD04>  { [         r,            R, registered ] };
+    key <AD05>  { [         t,            T, trademark ] };
+    key <AB03>  { [         c,            C, copyright ] };
+    key <AB08>  { [     comma,         less,  degree ] };
+    include "level3(ralt_switch)"
+};
+
-- 
1.6.0.4

